src/toolchain/toolchain.ts

```diff
@@ -46,6 +46,11 @@ interface SwiftTargetInfo {
     [name: string]: string | object | undefined;
 }
 
+export enum DarwinCompatibleTarget {
```

Comment:
do we want to support building for the simulator as well as device.

Reply:
I wasn't sure about how to expose the simulator, hence why I didn't add it to start. Do you think it should be (in the switcher UI), labeled something like `iOS (Simulator)`? I have not personally written any code that required simulator guards so I can't speak for how useful this is.

Reply:
We can give this a miss just now

Reply:
Sorry, I'm not familiar with that phrase 😓. Does "give this a miss" (here and other instances in this PR) mean to not worry about it for now?

Reply:
"give this a miss" means "lets not bother". We can look at this later. I know @stevapple is looking at supporting `destination.json` files, which could clean this up.

Reply:
"give this a miss" means "lets not bother"